services/tables/src/main/java/com/linkedin/openhouse/tables/model/TableDto.java

```diff
@@ -104,4 +104,13 @@ public Map<String, String> convertToMap() {
   private boolean isTableDtoPrimitive(Field field) {
     return field.getType() == String.class || field.getType() == long.class;
   }
+
+  /**
+   * Returns the primary key for TableDto Class
+   *
+   * @return TableDtoPrimaryKey
+   */
+  public TableDtoPrimaryKey getPrimaryKey() {
```

Comment:
can you add a method in `TableMapper.toTableDtoPrimaryKey(tableDto)` thats the singular place where we do such conversions

Reply:
Nice point. Made the change